src/plugins/intel_gpu/tests/test_cases/scatter_elements_update_gpu_test.cpp

```diff
@@ -90,3 +90,297 @@ TEST(scatter_elements_update_gpu_fp16, d2411_axisF) {
+    switch(format) {
+        case format::bfyx:
+            return "bfyx";
+        case format::b_fs_yx_fsv16:
+            return "b_fs_yx_fsv16";
+        case format::b_fs_yx_fsv32:
+            return "b_fs_yx_fsv32";
+        case format::bs_fs_yx_bsv16_fsv16:
+            return "bs_fs_yx_bsv16_fsv16";
+        case format::bs_fs_yx_bsv32_fsv16:
+            return "bs_fs_yx_bsv32_fsv16";
+        case format::bs_fs_yx_bsv32_fsv32:
+            return "bs_fs_yx_bsv32_fsv32";
+        case format::bfzyx:
+            return "bfzyx";
+        case format::b_fs_zyx_fsv16:
+            return "b_fs_zyx_fsv16";
+        case format::bs_fs_zyx_bsv16_fsv16:
+            return "bs_fs_zyx_bsv16_fsv16";
+        case format::bfwzyx:
+            return "bfwzyx";
+        default:
+            return std::to_string(format);
+    }
+}
+
+struct PrintToStringParamName {
+    template<class T>
+    std::string operator()(const testing::TestParamInfo<ScatterElementsUpdateParamsWithFormat<T> > &param) {
+        std::stringstream buf;
+        ScatterElementsUpdateParams<T> p;
+        format::type plain_format;
+        format::type target_format;
+        std::tie(p, plain_format, target_format) = param.param;
+        buf << "_axis=" << p.axis
+            << "_data=" << p.data_tensor.to_string()
+            << "_indices=" << p.indices_tensor.to_string()
+            << "_plainFormat=" << toString(plain_format)
+            << "_targetFormat=" << toString(target_format);
+        return buf.str();
+    }
+};
+}; // namespace
+
+template<typename T>
+struct scatter_elements_update_gpu_formats_test
+        : public ::testing::TestWithParam<ScatterElementsUpdateParamsWithFormat<T> > {
+public:
+    void test() {
+        const auto data_type = type_to_data_type<T>::value;
+        ScatterElementsUpdateParams<T> params;
+        format::type plain_format;
+        format::type target_format;
+
+        std::tie(params, plain_format, target_format) = this->GetParam();
+
+        auto& engine = get_test_engine();
+        const auto data = engine.allocate_memory({data_type, plain_format, params.data_tensor});
+        const auto indices = engine.allocate_memory({data_type, plain_format, params.indices_tensor});
+        const auto updates = engine.allocate_memory({data_type, plain_format, params.indices_tensor});
+
+        set_values(data, params.data);
+        set_values(indices, params.indices);
+        set_values(updates, params.updates);
+
+        topology topology;
+        topology.add(input_layout("Data", data->get_layout()));
+        topology.add(input_layout("Indices", indices->get_layout()));
+        topology.add(input_layout("Updates", updates->get_layout()));
+        topology.add(reorder("DataReordered", "Data", target_format, data_type));
+        topology.add(reorder("IndicesReordered", "Indices", target_format, data_type));
+        topology.add(reorder("UpdatesReordered", "Updates", target_format, data_type));
```

Comment:
Not sure what "different layouts" means here. If it is about formats, AFAIK all inputs of operation must have the same format. I.e., it is impossible to pass indices in ```bfyx``` and updates in ```b_fs_yx_fsv16```. 
If it is about dimensions, say, indices ```(2, 3, 4, 5)``` and updates ```(5, 6, 7, 8)``` - this would contradict operation specification - _"updates tensor of shape equal to the shape of indices tensor"_, see [https://docs.openvino.ai/latest/openvino_docs_ops_movement_ScatterElementsUpdate_3.html](https://docs.openvino.ai/latest/openvino_docs_ops_movement_ScatterElementsUpdate_3.html)

Reply:
>  I.e., it is impossible to pass indices in bfyx and updates in b_fs_yx_fsv16.

Why? I think it's not really prohibited on the graph level

Reply:
Added support for mixed input formats.